browser/src/core/Socket.js

```diff
@@ -1516,6 +1516,14 @@ app.definitions.Socket = L.Class.extend({
 
 	_onSocketClose: function () {
 		window.app.console.debug('_onSocketClose:');
+		if (!this._map._docLoadedOnce) {
+			var postMessageObj = {
+				success: false,
+				errorMsg: errorMessages.websocketconnectionfailed
```

Comment:
Any chance we could have a static identifier on the error type? With the current way we can still not render the link properly within the richdocuments error screen as.

I imagined something like this where any possible error type is also documented along the Action_Load_Resp in the sdk docs.

```suggestion
				errorMsg: errorMessages.websocketconnectionfailed,
				errorType: 'websocketconnectionfailed'
```

Reply:
With this we could actually decide on the error message in richdocuments without the need to pass around post messages (while i think this still makes sense for other cases to customize errors displayed within Collabora).

Just knowing the type of error allows to decide on different messages for admins or regular users and also be more flexible to show the url for example as a call to action button instead of the raw link.